tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js

```diff
@@ -0,0 +1,13 @@
+export function sumFibs(num) {
```

Comment:
as follow up to fibo look how you can memoize: https://medium.com/@reallygordon/implementing-memoization-in-javascript-5d140bb04166